client/signup/config/flows-pure.js

```diff
@@ -201,13 +201,8 @@ export function generateFlows( {
 			optionalDependenciesInQuery: [ 'coupon' ],
 			props: {
 				plans: {
-					/**
-					 * This intent is geared towards customizations related to the paid media flow
-					 * Current customizations are as follows
-					 * - Show only Personal, Premium, Business, and eCommerce plans (Hide free, enterprise)
-					 */
-					intent: 'plans-paid-media',
```

Comment:
@southp This is the change that caused the breakage in https://github.com/Automattic/martech/issues/3096. I am guessing `intent` was removed by mistake - or was there any other reason?

Reply:
also - FYI - removing an `intent` should be accompanied by a change to the [PlansIntent type](https://github.com/Automattic/wp-calypso/blob/trunk/packages/plans-grid-next/src/types.ts#L69). It's not enough to just root it out from the consuming end alone.

Reply:
Closing the loop, further discussion is happening in https://github.com/Automattic/wp-calypso/pull/92112#pullrequestreview-2144715592 :)